mapper.go

```diff
@@ -58,7 +59,7 @@ func (m *metricMapper) initFromString(fileContents string) error {
 
 		switch state {
 		case SEARCHING:
-			if line == "" {
+			if line == "" || line[0] == COMMENT_CHAR {
```

Comment:
I think it'd make more sense to allow these on all lines, not just between metrics.